.github/workflows/release-nightly.yml

```diff
@@ -40,3 +40,24 @@ jobs:
         run: yarn run publish:nightly
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Get latest tag
+        uses: actions-ecosystem/action-get-latest-tag@v1
```

Comment:
ain't this loading latest git tag?

Nightly releases don't have tag.

You could do something like:
```
runs: |
   packageJson=`cat ./package.json`
   version=`echo $(jq -r '.version' <<< "$packageJson")`
   echo "::set-output name=version::$version"
```
not sure if this works needs testing 😅